src/za/ac/sun/cs/green/service/canonizer/SATCanonizerService.java

```diff
@@ -59,12 +59,13 @@ public Expression canonize(Expression expression,
 		try {
 			log.log(Level.FINEST, "Before Canonization: " + expression);
 			invocations++;
-			OrderingVisitor orderingVisitor = new OrderingVisitor();
-			expression.accept(orderingVisitor);
-			expression = orderingVisitor.getExpression();
+//			OrderingVisitor orderingVisitor = new OrderingVisitor();
```

Comment:
Please remove unused code, rather than just uncommenting. And maybe make use of Javadoc to make things a bit easier to make sense of.